Xamarin.Forms.GoogleMaps/Xamarin.Forms.GoogleMaps.iOS/Logics/PinLogic.cs

```diff
@@ -59,6 +59,7 @@ protected override Marker CreateNativeItem(Pin outerItem)
             nativeMarker.GroundAnchor = new CGPoint(outerItem.Anchor.X, outerItem.Anchor.Y);
             nativeMarker.Flat = outerItem.Flat;
             nativeMarker.ZIndex = outerItem.ZIndex;
+            nativeMarker.Opacity = 1 - outerItem.Transparency;
```

Comment:
Why on this line is 1 - outerItem.Transparency;
and on line 318 1f - outerItem.Transparency; ?

Reply:
Transparency(used for Android API) means 0-visible and 1-invisible.
Opacity(used for iOS SDK) means 0-invisible and 1-visible.

GoogleMaps Android API and GoogleMaps iOS SDK uses "Transparency", "Opacity", "Alpha" it is so complexity.
So I use only "Transparency" in Xamarin.Forms.GoogleMaps it's simply.

Reply:
I mean in this file on iOS once you use 1f - ... and once 1- why "f" is missing?

Reply:
aha. Yes, missing "f".
But maybe no problem because ``outerItem.Transparency`` is float.
``int`` - ``float`` is ``float(or double?)``.

Reply:
I already fixed in latest source. :sweat_smile:

https://github.com/amay077/Xamarin.Forms.GoogleMaps/blob/master/Xamarin.Forms.GoogleMaps/Xamarin.Forms.GoogleMaps.iOS/Logics/PinLogic.cs#L62